install/update_91_92.php

```diff
@@ -1039,7 +1039,9 @@ function update91to92() {
       //Just display a Warning to the user.
       $migration->displayWarning("An index must be added in the 'id_search_option' field " .
          "of the 'glpi_logs table'; but your glpi_logs table is " .
-                                 "too huge. You'll have to add it on your database.");
+         "too huge. You'll have to add it on your database " .
+         "with the following query: " .
```

Comment:
This message will be displayed in console (cliupdate.php - the way I use) and indeed in HTML. On the HTML page, there are chances this will not change anything, but it will certainly in cli mode.